ROADMAP.md

```diff
@@ -0,0 +1,38 @@
+# KF Serving 2019 Roadmap
+## Q2 2019
+
+### Core CUJs
+* High Level Interfaces
+    * Deploy a Tensorflow model without specifying a Tensorflow Serving Technology.
+    * Deploy a XGBoost model without specifying a XGBoost Serving Technology.
+    * Deploy a ScikitLearn model without specifying a ScikitLearn Serving Technology.
+    * Deploy a Custom Containerized model by specifying.
+
+* Model Rollout
```

Comment:
eg. for this one could be: Create a minimal support for experimenting and rolling a new model to an existing production deployment.

Reply:
ack